### ISSUE TITLE
Plugin is not Open Source

### ISSUE THREAD

**Issue by ellbristow** [dwilson390/MyChunk#5]
As the owner of the code for the original version of this plugin, I would ask that you make this repository private, or take it offline.

This plugin is no longer available on an OpenSource licence and is the intellectual property of Elliott Bristow (ellbristow). Sharing of the code in a public repository is considered breach of the original copyright.

Whilst I do not INTEND on taking any legal action as a result of this breach, I would ask that you respect the copyright in place and take the repository offline.

You MAY use this code for the development of a private version of this plugin for use on your own server, but the resulting code may not be distributed publicly if it contains any of the original code.

Thank you for your compliance.

**Comment by DarkSeraphim** [comment]
@ellbristow he forked it under an open source license, thus this code still falls under that license. You changing the license won't change that, and I wish you the best of luck convincing legal authorities otherwise.

In order for your license change to be effective, he most likely has to agree with the license again, by using the 'new' (as far as you can call it new) code. As long as he does not do that, he does not have to comply with your new license.

Have a nice day,
DarkSeraphim

(I assume you could call this a copyright troll)

**Comment by ellbristow** [comment]
@DarkSeraphim I wish you hadn't said that. I didn't want to do it this way...

Per the GPLv3 licence this was originally distributed under:

"You may convey a work based on the Program, or the modifications to produce it from the Program, in the form of source code under the terms of section 4, provided that you also meet all of these conditions:

a) The work must carry prominent notices stating that you modified it, and giving a relevant date.
b) The work must carry prominent notices stating that it is released under this License and any conditions added under section 7. This requirement modifies the requirement in section 4 to “keep intact all notices”."

The plugin.yml file has removed my name as author and replaced it with @dwilson390 's. At no point does the plugin.yml, readme file or any other area of the code provide attribution that I was the original developer. This is a breach of the licence therefore authorising me to revoke it.

In accordance with section 8 of the GPLv3 licence I hereby terminate the licence in relation to the code used to form this git repository.

Citation: http://www.gnu.org/licenses/gpl-3.0.en.html

With that said, as the copyright holder I hereby authorise the use of this code for private development, on the provision that details of the original code are not made publicly available.

**Comment by DarkSeraphim** [comment]
@ellbristow the license will persist if he cures the violation within 30 days after the first notice from the original author, also under the section of Termination. That gives him a full month to change that, which I am sure is plenty of time.
> Moreover, your license from a particular copyright holder is reinstated permanently if the copyright holder notifies you of the violation by some reasonable means, this is the first time you have received notice of violation of this License (for any work) from that copyright holder, and you cure the violation prior to 30 days after your receipt of the notice.
Make sure you read the full section before quoting from it.

**Comment by ElieTGM** [comment]
@DarkSeraphim , @ellbristow , @dwilson390 
It's just a plugin..
Gosh..

**Comment by t7seven7t** [comment]
Regardless, @ellbristow you never published any of your source here to github with any license so your source is technically unlicensed. However by publishing it to Github you have agreed to their terms that:

_We claim no intellectual property rights over the material you provide to the Service. Your profile and materials uploaded remain yours. However, by setting your pages to be viewed publicly, you agree to allow others to view your Content. **By setting your repositories to be viewed publicly, you agree to allow others to view and fork your repositories.**_

https://help.github.com/articles/github-terms-of-service/

**Comment by DarkSeraphim** [comment]
@ElieTGM just trying to help a developer which gets some nice legal threats thrown at him ;). Things can appear to be really scary when you don't know the extent of the laws

**Comment by ellbristow** [comment]
@DarkSeraphim  Indeed, if he makes the necessary changes within 30 days the licence will be provisionally reinstated. I am well aware of the clause, I was merely stating the grounds under which the licence is being revoked, thereby providing notification via reasonable means.

Please, note from my OP, I am not throwing any legal threats. I stated I do NOT intend on taking any legal action, I simply would like my intellectual property preserved, and proper attribution given.

@t7seven7t Please... my code was published on Github for many years. This was forked from my original repository. My name is shows as contributor on every file. The fact that the repo is now private, and in fact stored at a different location, does not mean it was "never published.. here to github".

Indeed it was originally published on github using GPLv3, and therefore was forkable, under GPLv3... this is no longer the case. Github themselves claim no intellectual property, however I as the developer do, especially in cases where the source is made to look like I was not the original author.

**Comment by dwilson390** [comment]
I completely understand @ellbristow I wholeheartedly agree your intellectual property should be preserved.

Would it be possible to contact me via email at contact@dwilson.me.uk to sort this issue out in private. I would like to make a proposal.

Many thanks, and my sincere apology for the way this issue has turned out.

**Comment by DarkSeraphim** [comment]
@ellbristow hope that fixes a few of your issues listed in one of your posts

**Comment by ellbristow** [comment]
Thank you @dwilson390. I have sent you an email... though I suggest you remove your email address from this public forum :) I have it now.

@DarkSeraphim I am now addressing this issue in a civilised fashion via email. Thank you for your input.
